doc/src/Thumbnails.dox

```diff
@@ -0,0 +1,57 @@
+/*!
+ * @page thumbnails Thumbnails
+ *
+ * @section thumbnails_overview Overview
+ *
+ * OAIO provides a mechanism for a @ref manager to request a thumbnail
+ * from the @ref host when a new @ref entity is created, and for a host
+ * to look up a thumbnail for an existing entity.
+ *
+ * @section thumbnails_creation_during_publish Requesting Thumbnail Creation During Publishing
```

Comment:
[consider][minor] This title gets cut off (well, causes a horizontal scrollbar) in the sidebar. Very minor quibble.

Reply:
Saving this for later, as we have other long titles.